crates/weaver_semconv/src/group.rs

```diff
@@ -86,6 +87,9 @@ pub struct GroupSpec {
     pub name: Option<String>,
     /// The readable name for attribute groups used when generating registry tables.
     pub display_name: Option<String>,
+    /// The event body definition
+    /// Note: only valid if type is event
+    pub body: Option<BodySpec>,
```

Comment:
Do you have any thoughts on how to prevent somebody from adding body on the existing 'span events' definitions beyond code review?

Also, assuming we have smart codegen for events, what should it do? Generate span event? log event? Doesn't matter? Not possible until we deprecate span-events?

Reply:
Looks like a good candidate for the policy engine.

Reply:
The intent is that an `event` is an `event`, we would not want to "stop" someone adding `body field` to any `event` definition.

The `body` is optional, so as long as the new definition is defined as optional then (today) the same `event` definition can be used for both log and span events, on the proviso that it's currently not possible to pass the body fields as a `span` event.

One possible way of forward dealing with this would be something like https://github.com/open-telemetry/opentelemetry-specification/issues/3406#issuecomment-1523629641 where the `log.body` attribute would be defined as a JSON encoded string.